src/main/resources/registries/registry_key_class_relation.json

```diff
@@ -1,30 +1,31 @@
 {
+	"minecraft:attribute": "org.bukkit.attribute.Attribute",
+	"minecraft:banner_pattern": "org.bukkit.block.banner.PatternType",
+	"minecraft:block": "org.bukkit.block.BlockType",
+	"minecraft:cat_variant": "org.bukkit.entity.Cat$Type",
+	"minecraft:damage_type": "org.bukkit.damage.DamageType",
+	"minecraft:enchantment": "org.bukkit.enchantments.Enchantment",
 	"minecraft:entity_type": "org.bukkit.entity.EntityType",
+	"minecraft:fluid": "org.bukkit.Fluid",
+	"minecraft:frog_variant": "org.bukkit.entity.Frog$Variant",
+	"minecraft:game_event": "org.bukkit.GameEvent",
+	"minecraft:instrument": "org.bukkit.MusicInstrument",
+	"minecraft:item": "org.bukkit.inventory.ItemType",
+	"minecraft:jukebox_song": "org.bukkit.JukeboxSong",
+	"minecraft:map_decoration_type": "org.bukkit.map.MapCursor$Type",
+	"minecraft:memory_module_type": "org.bukkit.entity.memory.MemoryKey",
+	"minecraft:menu": "org.bukkit.inventory.MenuType",
```

Comment:
Only this line is new, all the others have the same value, just have been sorted.

Reply:
Is this sorting something new, or will it cause large diffs like this in the future? If so, we should stabilize it.